oci/oci.go

```diff
@@ -766,6 +766,9 @@ func PrepareProcessExec(c *Container, cmd []string, tty bool) (*os.File, error)
 
 	pspec := c.Spec().Process
 	pspec.Args = cmd
+	// We need to default this to false else it will inherit terminal as true
+	// from the container.
+	pspec.Terminal = false
```

Comment:
mmm that's why we changed it in the first place to inherit, do we _always_ want this to be `false`?

Reply:
We can let the caller decide if they want to inherit and pass it down explicity as true.

Reply:
alrighty, figured, thanks!